DEVELOPMENT.md

```diff
@@ -98,7 +98,8 @@ Due to backwards compatability of the plugin, the Jenkins version defined in `po
 ```
 ...
 <properties>
-    <java.level>8</java.level>
+    <maven.compiler.source>1.11</maven.compiler.source>
+    <maven.compiler.target>1.11</maven.compiler.target>
```

Comment:
Currently neither `<java.level>8</java.level>` nor `<maven.compiler.*>` tags are present in the plugin's `pom.xml`.
The former was deprecated, and the latter are simply not needed: starting with JDK 9 the [maven compiler plugin](https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html) uses `<maven.compiler.release>` property. Moreover, the HPI plugin [sets this property automatically](https://github.com/jenkinsci/maven-hpi-plugin/blob/47b996d185d0d209032289a275839b50fb4d5150/src/main/java/org/jenkinsci/maven/plugins/hpi/ValidateMojo.java#L46). The bottom line is, these two lines simply need to be removed.

Reply:
Done.